content/participants/ttcj@rahul1116/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Rahul Ravindran
+type: "participant"
+name: "Rahul Ravindran"
+portfolio: "null"
+github-link: "https://github.com/rahul1116"
+linkedin-link: "https://www.linkedin.com/in/rahulravindran11/"
+looking-for: "job"
+description: "Very good in HTML and CSS. I have good understanding of React and Node.js."
+---
+
+Hi! I am a Rahul Ravindran. Let me walk you through my journey so far.
+
+### introduction
+
+I am passionate about JavaScript and its frameworks.
+
+## why are you taking this challenge?
+
+I need to learn and make good projects, write blogs and get recommendations from industrial experts, getting feedbacks and improve more with an wonderfull opportunity to get a job. This is a win win situation.
+
+## something interesting about you?
+
+I love the game Counter-Strike because if anyone asks me "Am I from CS background?", I can say yes I am from CS background(Counter_Strike ;) ).
+
+### help I need?
+
+Usage and best practices of using GraphQl in the project. I would also like to know more about integrating Google, Facebook, etc auth in my applications.
```

Comment:
these are good topics for help. Mentors can help you. Can you create two issues in the repo for this? Thanks in advance.

Reply:
sure